md2notion/NotionPyRenderer.py

```diff
@@ -24,6 +27,10 @@ class NotionPyRenderer(BaseRenderer):
     object containing a descriptor for every row. This is still TODO
     """
 
+    def __init__(self, *extras):
```

Comment:
Renaming `extras` to `extraExtensions` and also documenting it in a Docstring + in the README.md might be able to resolve #20